app/cdap/components/PipelineDetails/RunLevelInfo/index.js

```diff
@@ -54,8 +58,12 @@ export default function RunLevelInfo() {
             <RunStatus />
             <RunStartTime />
             <RunDuration />
-            <ConnectedRunNumWarnings />
-            <ConnectedRunNumErrors />
+            {shouldShowErrorAndWarningCounts && (
+              <React.Fragment>
+                <ConnectedRunNumWarnings />
+                <ConnectedRunNumErrors />
+              </React.Fragment>
```

Comment:
Current change-set will not show error snack bar if the error classification api response is empty, also we are removing Error and Warning info as per above code. Then what is the indication of error for end user when a pipeline run fails and classification api response is empty?

Reply:
Have accounted for this in the latest set of changes.

Reply:
nit: can you please add a screenshot for this too?

Reply:
Update: after another discussion, we decided not to use the error count from metrics in the snackbar, as this value does not represent the actual number of errors / exceptions. So, in this specific case, the message is "Pipeline execution failed.     [VIEW LOGS]"

Added the screenshot.